docs/installation.md

```diff
@@ -1,27 +1,30 @@
 # Installation
 
-
-This is a 3.2 Django project, running on MySQL, and using RabbitMQ as a message queue, and Redis for caching.
-
-It is tested on OS X and linux systems.
-
-To install it, create a virtual environment, and install the required packages.
-
-```
-make venv
-. venv/bin/activate
-pip install pipenv
-pipenv install
-```
-
-### Set up the environment
-
-By default, pipenv loads the contents of `.env` for any commands like opening a shell with `pipenv shell`, or running one off commands with `pipenv run command_name`.
-
-Copy the sample environemnt file at `.env.sample` to `.env`, and fill in the necessary environment variables:
-
-In production the admin_portal project relies on object storage for storing static files, but in development you may not need this.
-
-
-
-
+This installation shows you how to setup the Admin Portal of The Green Web Foundation.<br>
+In order to setup this project, we recommend following the guides from top to bottom. Skipping some topics might result in missing crucial steps for the project to work.
+
+---
+## Gitpod
+### Why Gitpod
+To speed up and simplify the installation process, we use Gitpod. <br> We strongly recommend using this tool.
+ 
+### Preparing a Gitpod workspace
+After logging into Gitpod or creating an account we recommend also installing the Gitpod browser extension. This extension integrates extra functionality in websites such as Github.
+
+1. Go to our [Github repository](https://github.com/thegreenwebfoundation/admin-portal) and click on the Gitpod button if you have the browser extention installed or manually create and link it in your Gitpod Workspaces. It might take a couple of minutes to prepare the environment.
+2. After Gitpod is finished preparing the environment, it will open the workspace. Close any unnecessary terminal windows. To open a new fresh terminal window press ```Ctrl + ` ```.
+---
+## Virtual environment
+In an isolated virtual environment we are able to act on various usefull tools within the project.
+
+### Setup
+Before following the following list, make sure you are in the root directory (workspace/admin-portal).
+1. Start by creating a virtual environment named *venv* by executing the following command in a terminal window: ```make venv```.
```

Comment:
I'm not sure you need three backticks. I think you can just use one at the start, and one at the end.